bootstrap.php

```diff
@@ -459,6 +459,31 @@ function ($app) {
     }
 );
 
+$app['place_history_projector'] = $app->share(
```

Comment:
Nit: Ideally these services would be moved to a service provider. Would also make it a bit easier to get a better overview when porting away from Silex.